app/models/katello/repository.rb

```diff
@@ -968,7 +968,7 @@ def remove_docker_content(manifests)
       property :docker_upstream_name, String, desc: 'Returns name of the upstream docker repository'
     end
     class Jail < ::Safemode::Jail
-      allow :name, :label, :docker_upstream_name
+      allow :name, :label, :docker_upstream_name, :content_counts
```

Comment:
Sorry I'm late to the party, but any allowed method in `Jail` should be documented...